### ISSUE TITLE
Dev Container CLI fails on Windows - Empty executable not found in PATH

### ISSUE THREAD

**Issue by rzhao271** [microsoft/vscode-remote-release#5670]
Windows_NT x64 10.0.19043
VS Code: 1.61.0-insider (user setup) a4342f3addc29f405a5cb4aab359b0b98223411e
Remote-containers v0.200.0

I tried going through the steps in https://github.com/microsoft/vscode-remote-release/issues/5648#issue-1010334143, and after running `devcontainer-insiders build --image-tag feature-test .`, get the output

`findLocalWindowsExecutable: Exectuable '' not found on PATH`.

The output right before is 

```
[258 ms] Start: Run:  build -f c:\Users\raymondzhao\work\emmet-work\testworkspace3\.devcontainer\Dockerfile -t vsc-testworkspace3-74e4622d6eba1697b4a1d30cda8ec91e --build-arg VARIANT=16-bullseye c:\Users\raymondzhao\work\emmet-work\testworkspace3\.devcontainer
```

**Comment by Chuxel** [comment]
On windows I see something slightly different:

```
TypeError: _n is not a constructor
    at doe (c:\Users\clantz\.vscode-insiders\extensions\ms-vscode-remote.remote-containers-0.200.0\dev-containers-user-cli\dist\node\devContainersUserCLI.js:279:13815)
```

In my case, the source code is on E: if that matters

**Comment by chrmarti** [comment]
@rzhao271 This looks like you have the `"remote.containers.dockerPath"` setting set to the empty string. Fixing this by using the default `docker` in this case (aligning with the extension).

@Chuxel That's a minifier bug hiding an underlying issue. Fixing this by logging the underlying error, this will make the minifier issue go away in this case. That might resurface elsewhere (also updating the minifier, but I don't see relevant bug fixes since the version we've used for the current release).

**Comment by chrmarti** [comment]
@rzhao271 Could you verify with Remote-Containers 0.201.0?

**Comment by chrmarti** [comment]
@Chuxel Could you open a new issue with the new error that should be logged now?

**Comment by Chuxel** [comment]
Raised https://github.com/microsoft/vscode-remote-release/issues/5675 which is what I see now

**Comment by rzhao271** [comment]
Setting the setting to `docker` works, but `` doesn't work still, with the same error.

**Comment by chrmarti** [comment]
@rzhao271 Thanks, missed one case. Fixed in Remote-Containers 0.202.0.
